### PR TITLE
[bitnami/harbor] Update upgrade notes 'to 13.0.0'

### PR DESCRIPTION
**Pull request by gongomgra**
Signed-off-by: Gonzalo Gomez Gracia <gonzalog@vmware.com>

### CODE REVIEW

File: charts/harbor/_upgrade.md.erb [hunk 1]
```diff
@@ -9,6 +9,65 @@ weight: 20
 highlight: 20
 =end %>
 
+### To 13.x.x
+
+This major release updates the PostgreSQL subchart image to the newest 13.x version.
+
+#### Upgrading Instructions
+
+To upgrade to *13.x.x* from *12.x*, it should be done reusing the PVC(s) used to hold the data on your previous release. To do so, follow the instructions below (the following example assumes that the release name is `harbor`):
```
Comment (joancafom) [inline charts/harbor/_upgrade.md.erb:18]:
```suggestion
The upgrade process to *13.x.x* from *12.x.x* should be done by reusing the PVC(s) used to hold the data on your previous release. To do so, follow the instructions below (the following example assumes that the release name is `harbor`):
```

File: charts/harbor/_upgrade.md.erb [hunk 2]
```diff
@@ -9,6 +9,65 @@ weight: 20
...
+
+> NOTE: Please, create a backup of your database before running any of those actions.
```
Comment (joancafom) [inline charts/harbor/_upgrade.md.erb:20]:
```suggestion
> NOTE: Please, create a backup of your database before running any of these actions.
```

File: charts/harbor/_upgrade.md.erb [hunk 3]
```diff
@@ -9,6 +9,65 @@ weight: 20
...
+> **NOTE:** the instructions above reuse the same PostgreSQL version you were using in your chart release. Otherwise, you will find an error such as the one below when upgrading since the new chart major version also bumps the application version. To workaround this issu
+ you need to upgrade database, please refer to the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/upgrading.html) for more information about this.
+>
+> ```console
+> $ kubectl --namespace "${HARBOR_NAMESPACE:?}" logs harbor-postgresql-0 --container harbor-postgresql
+>     ...
+> postgresql 08:10:14.72 INFO  ==> ** Starting PostgreSQL **
+> 2022-02-01 08:10:14.734 GMT [1] FATAL:  database files are incompatible with server
+> 2022-02-01 08:10:14.734 GMT [1] DETAIL:  The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 13..
+> ```
```
Comment (joancafom) [inline charts/harbor/_upgrade.md.erb:69]: I would personally put this section first and tell users that they should follow the official notes to dump & upgrade their database scheme. Once warned, we can tell there is a workaround to continue using the old image (the steps described in the guide). WDYT?

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your PR @gongomgra ! I left you some comments, could you please take a look at them?

File: charts/harbor/_upgrade.md.erb (hunk 3 above)
Comment (gongomgra) [reply charts/harbor/_upgrade.md.erb]: This is the current guide format for previous versions, so I prefer to be consistent.

Comment (joancafom) [reply charts/harbor/_upgrade.md.erb]: 👍 Fair enough !

**Review by joancafom — APPROVED** [review]
LGTM

**Review by carrodher — APPROVED** [review]

**Pull request merged**